package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "lodash.assign": "^4.2.0",
     "lodash.intersection": "^4.4.0",
+    "lodash.isplainobject": "^4.0.6",
```

Comment:
Each of these appear be versioned separately. Look at all the different [version numbers](https://www.npmjs.com/browse/keyword/lodash-modularized). Looks like lodash though now supports you pulling out just the modules you want with just 
```coffee
require 'lodash/isPlainObject'
```
Looking at the lodash roadmap it appears they plan on deprecating these modules in favor of just doing that. I'll update lodash separately to that and then update this PR